### PR TITLE
How to create a quick React and Angular app with JHipster Lite

### PR DESCRIPTION
**Pull request by dsharpe**

### CODE REVIEW

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+JHipster Lite does not currently generate entities or CRUD pages for the front-end. The idea is to set you up with a relatively blank slate with some initial structure. This provides a good starting pattern that can be followed as the application grows.
+
+JHipster Lite also generates code that follows a [Hexagonal Architecture](https://alistair.cockburn.us/hexagonal-architecture/). This helps with designing a system around the business intent.
+
+# Let’s Build a React App
+
+## Run JHipster Lite locally, if you want
+
+You can certainly go straight to the [JHipster Lite website](https://lite.jhipster.tech/landscape) and start generating there. But you can also run it locally if you have Docker. (or in my case, Rancher)
+
```
Comment (GospelBeats) [inline posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:35]: I suggest linking to [Docker](https://www.docker.com) and/or linking to how to install docker if the reader don't already have it installed (https://docs.docker.com/get-docker)

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+- React
+- Angular
+- Hexagonal Architecture
+date: 2022-12-13T12:21:50.000Z
+title: "How to Create a Quick React and Angular App with JHipster Lite"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2022/12/jhipster_lite_logo.png
+---
+# What is JHipster Lite?
+
+JHipster Lite is an application generator that generates step-by-step only what you need for a modern web application and/or microservice architecture. As you might imagine from the name, JHipster Lite is a different approach to application generation than its predecessor, [JHipster](https://www.jhipster.tech/).
```
Comment (GospelBeats) [inline posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:16]: Possible link opportunity for "application generator" for those who may never heard of one: https://www.pcmag.com/encyclopedia/term/application-generator

**Review by GospelBeats — APPROVED** [review]
Excellent blog! I enjoyed reading! I like the step by step follow-along!

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+import { getBeers } from '@/beers/services/beers';
+import { IBeer } from '@/beers/interface';
+ 
+export default function Beers() {
+   const [beers, setBeers] = useState<Array<IBeer>>([]);
+ 
+   useEffect(() => {getBeers({setBeers})}, []);
+ 
+   return (
+       <main style={{ padding: "1rem 0" }}>
```
Comment (codyfrenzel) [inline posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:131]: I would remove the inline styling. CSS styles don't really add anything to your code blocks other than clutter. Unless you wanted someone to literally be able to copy paste out of the blog and get it working.

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+
+Add a link to the new beers page
+
+```html
+<Link to="/beers">I love beer</Link>
+```
+
+## Add some beers with the service for display
+
+Restart your application to make sure you have the latest changes
```
Comment (codyfrenzel) [inline posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:191]: I would assume there is a live reload option to detect changes, no?

Comment (dsharpe) [reply posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:189]: I decided the blog was long enough so I did not want to get into that. You have to run the npm serve and hit the other port to get the live reload.

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+export class AppRoutingModule {}
+```
+
+The last step is to add a link to the home page at `app.component.html`
+
+```html
+ <a routerLink="beers" routerLinkActive="active" ariaCurrentWhenActive="page">I love beer</a>
+ ```
+
+ You can now restart and add some beers to the service using the steps above
```
Comment (codyfrenzel) [inline posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:438]: Same as the other comment. I would be very surprised if JHipster Lite didn't have live reload built in to monitor code changes in the projects. Should be able to "refresh and add some beers"

File: posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md (hunk 4 above)
Comment (codyfrenzel) [reply posts/how-to-create-a-quick-react-and-angular-app-with-jhipster-lite.md:189]: Ah okay, yeah I assume most FE people would understand. From me reading I'm like, "oh if I have to do that for every code change it is kind of a bad developer experience"

**Review by codyfrenzel — APPROVED** [review]

**Pull request merged**